src/pages/components/table-of-contents.mdx

```diff
@@ -54,75 +64,91 @@ The Table of contents (ToC) component renders as left navigation that includes a
-- The Table of contents needs to be used across alternating full width themed sections throughout a page.
+![Image of the vertical table of contents at tablet breakpoint](../../images/component/table-of-contents/toc-vertical-behavior-01.png)
 
-#### When not to use:
+<Caption>Example of vertical table of contents at the medium breakpoint</Caption>
 
-- The horizontal version should not be used with a center aligned lead space because the text alignment disrupts the user's reading flow. In that situation, consider using the vertical version with Lead space centered or Lead space block.
-- The horizontal version should not be placed immediately below the IBM masthead.
+</Column>
+
+<Column colMd={8} colLg={4}>
+
+![Image of the vertical table of contents at small breakpoint](../../images/component/table-of-contents/toc-vertical-behavior-02.png)
 
-## Modifiers
+<Caption>Example of vertical table of contents at the small breakpoint</Caption>
 
-### Vertical with heading content
+</Column>
+
+</Row>
 
-This variation only applies to the vertical Table of contents (ToC). The area above the Table of contents can be utilized in many ways—showing navigational links, the page title, or an image.
+## Horizontal
 
-![Image of the vertical table of contents with heading content](../../images/component/table-of-contents/table-of-contents--with-heading.png)
+The horizontal variation displays the table of content navigation links in a row. When present, it sticks to the top of the page while scrolling. It then replaces the Masthead sticky navigation when the top of its container meets the top of the window.
 
-<Caption>
-  Example of the vertical Table of contents with heading content.
-</Caption>
+![Image of the horizontal table of contents within a page](../../images/component/table-of-contents/toc-horizontal.png)
+
+<Caption>Example of the horizontal table of contents within a page.</Caption>
 
 #### When to use:
 
-- The vertical version can take up the first four columns of the 2x grid to help the user navigate related content.
+- Your experience needs to use full width section components or the first four columns of the 2x grid.
+- The Table of contents needs to be used across alternating full width themed sections throughout a page.
 
 #### When not to use:
 
-- The vertical version should not be used when any of the associated content components uses a different theme (for example, the page theme is White, but one of the table of contents components uses a Gray 100 theme). In that situation, consider using only one theme throughout the vertical table of contents or using the horizontal version instead.
-- The vertical version should not contain any components that use a section header.
+- The horizontal version should not be used with a center aligned lead space because the text alignment disrupts the user's reading flow. In that situation, consider using the vertical version with Lead space centered or Lead space block.
```

Comment:
```suggestion
- The horizontal version should not be used with a center aligned lead space because the text alignment disrupts the user's reading flow. In that situation, consider using the vertical ToC with lead space centered or lead space block.
```